source/configuration_manual/nfs.rst

```diff
@@ -30,8 +30,21 @@ NFS mount options
 * ``root_squash``: Dovecot doesn't care about this. Typically Dovecot doesn't
   store any root-owned files in NFS.
 
-* ``nolock``: This is possible to use either if ``lock_method=dotlock`` is
-  used, or as a slightly unsafe optimization (see below).
+* ``nolock``: This is possible to use as a slightly unsafe optimization.
```

Comment:
there is also local_lock=all option.

Reply:
local_lock=mechanism

Specifies whether to use local locking for any or both of the flock and the POSIX locking mechanisms. mechanism can be one of all, flock, posix, or none. This option is supported in kernels 2.6.37 and later.

The Linux NFS client provides a way to make locks local. This means, the applications can lock files, but such locks provide exclusion only against other applications running on the same client. Remote applications are not affected by these locks.

Reply:
Added. I think it's equivalent to nolock.